hoot-core/src/main/cpp/hoot/core/visitors/CalculateMapBoundsVisitor.h

```diff
@@ -64,7 +64,6 @@ class CalculateMapBoundsVisitor : public ElementConstOsmMapVisitor
 
 private:
 
-  const OsmMap* _map;
```

Comment:
This one the base class of `ElementConstOsmMapVisitor` has a member `_map` also so this one overwrites it and it isn't necessary.